MASH-FRET/source/mod-trace-processing/management/traceManager.m

```diff
@@ -2071,6 +2046,62 @@ function plotData_autoSort(h_fig)
 
 end
 
+function drawMask(h_fig,x,y,D)
+
+% defaults
+clr = [0.5,0.5,0.5];
```

Comment:
In the tabs "Auto sorting" all masked intensity/FRET regions are grayed out (i.e. everything not within xlow to xup). In the "Overview" tab the unselected molecules are grayed out too. This is intuitive. However in the upper panel of the "Overview" tab where the concatenated trace is displayed, the grayed area corresponds to the molecules currently in view. I suggest to do the following for consistency: Gray out the area in the concatenated trace corresponding to all "unselected" molecules and color the area of the molecules currently in view with a faint desaturated blue or similar (both in the concatenated traces and in the molecule selection panel). What do you think?

Reply:
Yes! I got the same idea! In MASH-code 6 the molecules in view are on a white background in the concatenated traces, and the molecules out of view ar masked in grey. I don't think adding another color (blue) would bring more help, to my opinion, white is sufficient (TM has already a lot of colors)

Reply:
Agreed!